components/lcd/esp_lcd_gc9503/include/esp_lcd_gc9503.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * @file
+ * @brief ESP LCD: GC9503
+ */
+
+#pragma once
+
+#include "esp_lcd_types.h"
+#include "esp_lcd_panel_rgb.h"
+
+#include "esp_lcd_panel_io_additions.h"
```

Comment:
Could we move this include to the .c file and make esp_lcd_panel_io_additions a private dependency?

Reply:
Yeah, I directly remove it.